tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -1524,4 +1524,218 @@ TestCarePartnerCallParticipantsDeviceCallIgnored:
       Asserts:
         - Type: eq
           Var: CALL_MESSAGE
-          Value: 'No active call.'
\ No newline at end of file
+          Value: 'No active call.'
+
+# CXTB-128: Call Participant's device - Call Missed.
+TestCarePartnerCallParticipantsDeviceCallMissed:
+  Vars:
+    MESSAGE_TO_PARTICIPANT: Automation Message CXTB-128
+  Environment: Settings
+  Precases:
+    - LoginNeverAloneHelper
+    - CarePartnerCleanCallQueueAndHangedCalls
+    - SeniorCleanPrompts
+  Aftercases:
+    - TerminateApp
+  Roles:
+    - Role: desktopChrome
+      App: desktop 
+    - Role: localiOS
+      Capabilities:
+        newCommandTimeout: 100
+      App: NeverAlone
+  Actions:
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # Navigate to Participants.
+    - Type: case
+      Value: CarePartnerNavigateToParticipants
+  # Click on the corresponding participant.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.senior_complete_name$
```

Comment:
Please check this page element, the component has changed.

Reply:
https://github.com/neveralone-chs-org/TestRay/pull/101 this PR contains the update for the element